packages/widget-todo/src/types.ts

```diff
@@ -30,6 +30,7 @@ export interface ContextValues extends Configuration, State {
 export interface Events {
   openAddTodoDialog: boolean
   openEditTodoDialog?: string
+  openCloudSyncFeatureInterest: boolean
```

Comment:
Let's add this to the `MarqueeEvents` interface so we only have to define it at one place